src/l500/l500-private.h

```diff
@@ -25,7 +25,7 @@ namespace librealsense
     {
         // L500 depth XU identifiers
         const uint8_t L500_HWMONITOR = 1;
-        const uint8_t L500_AMBIENT = 2;
+        const uint8_t L500_DIGITAL_GAIN = 2; // Renamed from L500_AMBIENT
```

Comment:
Should this be the same name as in FW? Do we know what it is?

Reply:
I tried asking Sami and he didn't understand what I needed, so I changed the name so we will know what does it do and kept the old name in the comment.